website/content/en/preview/concepts/deprovisioning.md

```diff
@@ -49,16 +47,29 @@ There are both automated and manual ways of deprovisioning nodes provisioned by
     # Delete all nodes owned by a specific provisioner
     kubectl delete nodes -l karpenter.sh/provisioner-name=$PROVISIONER_NAME
     ```
+* **Provisioner Deletion**: Nodes are considered to be "owned" by the Provisioner that launched them. Karpenter will gracefully terminate nodes when a provisioner is deleted.
```

Comment:
```suggestion
* **Provisioner Deletion**: Nodes are owned by the Provisioner through an [owner reference](https://kubernetes.io/docs/concepts/overview/working-with-objects/owners-dependents/#owner-references-in-object-specifications) that launched them. Karpenter will gracefully terminate nodes through cascading deletion when the owning provisioner is deleted.
```

"owned" shouldn't be in quotes. This _is_ actually what's happening.

Reply:
Oh I didn't write this, but I can change it :)